### PR TITLE
Add egg fried rice recipe

### PR DESCRIPTION
**Pull request by lesterhan**

### CODE REVIEW

**Review by anneconover — APPROVED** [review]

**Review by rebeccasiow — APPROVED** [review]
please greg please

File: src/pages/recipes/egg-fried-rice.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+date: "2021-06-25"
+categories: ["Recipes"]
+status: "published"
+---
+
+Deep Dive on Egg Fried Rice (蛋炒饭 dan chao fan)
+
+_by Lester Han, guest contributer_
+
+Egg fried rice is the measure of a Chinese home cook. When done well it is table-slappingly good. When it's done poorly I will still eat the whole dang thing doesn't matter. It's an easy and satisfying comfort food that goes with any meal of the day.
```
Comment (groman00) [inline src/pages/recipes/egg-fried-rice.md]: I really like this description

**Review by groman00 — CHANGES_REQUESTED** [review]
Can we tighten up the excerpt and move some of the description into the article body?  See screenshot:

Image 6-25-21 at 1 18 PM

**Comment by lesterhan** [comment]
> Can we tighten up the excerpt and move some of the description into the article body? See screenshot:


updated

**Review by groman00 — APPROVED** [review]

File: src/pages/recipes/egg-fried-rice.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Egg Fried Rice (蛋炒饭 dan chao fan) is the measure of a Chinese home cook. When done well it is table-slappingly good.
+
+<!-- excerpt end -->
+
+When it's done poorly I will still eat the whole dang thing doesn't matter. It's an easy and satisfying comfort food that goes with any meal of the day.
+
+There are many variants of this dish, feel free to switch it up. I don't have precise measurements but here's a video of me making it and eating it: https://www.youtube.com/watch?v=jn2r8DQwCLI
+
+_by Lester Han, guest contributer_
```
Comment (rebeccasiow) [inline src/pages/recipes/egg-fried-rice.md:16]: Cannot believe Lester gets creds for this and an immediate merge but https://github.com/groman00/romanwebnet-gatsby/pull/10/files !!!!!!!!!!!!!! does not get merged???????

File: src/pages/recipes/egg-fried-rice.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: 'Deep Dive on Egg Fried Rice'
```
Comment (rebeccasiow) [inline src/pages/recipes/egg-fried-rice.md:2]: SINGLE QUOTES???? R U KIDDING MEE

**Pull request merged**